server/routes/deleteObject.js

```diff
@@ -8,16 +8,17 @@ app.use(cors());
 app.use(bodyParser.json());
 import { minioClient } from '../minioConfig.js';
 import { get_user_bucket, delete_file } from '../Database_queries/queries.js';
+import { logger, log } from '../logger.js';
 
 router.post('/:url', async function (req, res) {
   let user = await get_user_bucket(req.user.user_email);
   let fileName = req.body.fileName;
-  let fileId = req.body.fileId;
   let miniopath = '/hv/' + user + '/thumbnail/';
   let bucketName = 'datadrive-dev';
   let format = fileName.split('.')[1];
   let name = fileName.split('.')[0];
-  let fileName_thumbnail = fileName.split('.')[0] + '.png' + fileId;
+  let fileName_thumbnail = fileName.split('.')[0] + '.png';
+
   minioClient.removeObject(
```

Comment:
The order of function calls should be adjusted to ensure that `delete_file` is called after `delete_user_socket`. This is important for maintaining the correct sequence of operations. 
```diff
-   await delete_file(user, bucketName, name);
+   await delete_file(user, bucketName, fileId);
```
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*